braid/braid_test.h

```diff
@@ -282,6 +308,32 @@ braid_TestDelta(braid_App               app,          /**< User defined App stru
                 braid_PtFcnStep         mystep        /**< Compute a time step with a braid_Vector */
                 );
 
+/**
+ * Warmup calls all user functions once with sensible arguments.
+ * This is especially useful for the Julia wrapper, since Julia is JIT compiled,
+ * and this force compiles all the functions used in braid_Drive.
+ */
+braid_Int
+braid_Warmup( braid_App                app,         /**< User defined App structure */
```

Comment:
All the other functions in `braid_test` are for doing unit testing.  Does Warmup belong here or should the name be changed?